example/package-lock.json

```diff
@@ -1,9 +1,36 @@
 {
```

Comment:
I think we still don't want to check-in/zip the example/package-lock.json, right?

Reply:
Right, I missed that, I think fixed when I pushed after the 4.0 release.